### DISCUSSION TITLE
Microsoft has spent the last year or so without success trying to acquire which massive gaming corporation?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#107] [Galaxy Trivia] [answered]
**Hint:** CoD

**Comment by Tynab** [comment]


**Reply by ParanoidUser** [reply]
Nice try, but "Discord" is not a game studio.

**Comment by Faiqlatheef** [comment, marked as answer]
Microsoft has been trying to acquire the gaming corporation called "Activision Blizzard" for the last year or so, but as of my knowledge cutoff of September 2021, the acquisition had not been completed. Activision Blizzard is the parent company of popular video game franchises like Call of Duty, World of Warcraft, and Candy Crush.

**Reply by ParanoidUser** [reply]
You did it! 😋 Congrats on your well deserved [Galaxy Brain](https://github.com/Faiqlatheef?tab=achievements&achievement=galaxy-brain) badge!
If you're willing to be a part of the history, feel free to open PR and add your name to the [change log](https://github.com/ParanoidUser/yolo/blob/main/docs/CHANGELOG.md).
